es/utils/aepp-wallet-communication/rpc/wallet-rpc.js

```diff
@@ -246,6 +246,18 @@ export const WalletRpc = Ae.compose(Accounts, Selector, {
     getClients () {
       return rpcClients
     },
+    /**
+     * Remove specific RpcClient by id
+     * @function removeClient
+     * @instance
+     * @rtype (id: string) => Boolean
+     * @param {String} id Client id
+     * @param {Object} [opt = {}]
+     * @return {Object}
+     */
+    removeClient (id, opt = { forceConnectionClose: false }) {
```

Comment:
```suggestion
    removeRpcClient (id, opt = { forceConnectionClose: false }) {
```
to keep it consistent with `addRpcClient`